bin/lib/yargs.js

```diff
@@ -58,7 +58,6 @@ module.exports = require('yargs')
     'e': {
       alias: 'endpoint',
       type: 'string',
-      default: 'https://api.github.com',
```

Comment:
think this should still be here as the default so that usage text includes the actual default